server/train/train_types.py

```diff
@@ -17,7 +17,7 @@
 CGROUP_FEATURES = ["cgroupfs_cpu_usage_us", "cgroupfs_memory_usage_bytes", "cgroupfs_system_cpu_usage_us", "cgroupfs_user_cpu_usage_us"]
 IO_FEATURES = ["bytes_read", "bytes_writes"]
 BPF_FEATURES = ["cpu_time"]
```

Comment:
can you add irq counters too?

Reply:
I think @KaiyiLiu1234 is working on that in another PR.

Reply:
Good thing I waited. I can include change as soon as this PR is merged. The prom queries are very clean now. For the irq counters, I believe we also need to set up the initial model as well.

Reply:
I will test it asap